src/commands/edge_app.rs

```diff
@@ -62,6 +74,13 @@ impl EdgeAppCommand {
 
         let manifest = EdgeAppManifest {
             app_id,
+            settings: vec![Setting {
+                title: "username".to_string(),
+                type_: "text".to_string(),
+                default_value: "stranger".to_string(),
+                optional: true,
+                help_text: "An example of a setting that is used in index.html".to_string()
+            }],
```

Comment:
Add a setting for the `screen_name` as well.

Reply:
screen_name is a part of metadata, not a setting

Reply:
Oh ok, thanks. Sorry, I thought of the other "screen name" (like an alias name in general). Got it.
LGTM 👍🏻